atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -66,7 +63,7 @@ public ServiceDiscoveringAtlasSupplier(
             LongSupplier timestampSupplier) {
```

Comment:
Not in scope (?) but the inversion of parameters here and in `atlasFactory.createRawKeyValueService` is infuriating

Reply:
Yeah, now's not a good time to touch this class's API (and above)